### PR TITLE
Ayr 769/mobile search tb results

### PR DESCRIPTION
**Pull request by colinbowen**
## Changes in this PR
- Add mobile search layout

## JIRA ticket
https://national-archives.atlassian.net/browse/AYR-769

## Screen shots of UI changes
Screenshot 2024-03-19 at 11 56 18

Screenshot 2024-03-19 at 11 56 24

### After
Screenshot 2024-03-19 at 11 54 39
Screenshot 2024-03-19 at 11 54 47

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

File: app/tests/test_search.py [hunk 1]
```diff
@@ -18,6 +18,7 @@ def verify_search_transferring_body_header_row(data):
     expected_row = (
         [
             "Series",
+            "Series / File name / Consignment ref",
```
Comment (anthonyhashemi) [inline app/tests/test_search.py]: so we expect to see words series, file name and consignment twice?

Comment (colinbowen) [reply app/tests/test_search.py]: We're using CSS to only display one or the other for Desktop / Mobile. The HTML does contain both headers though to account for this.

Comment (anthonyhashemi) [reply app/tests/test_search.py]: so does it look funky if user has css disabled?

Comment (pgandhizaizi) [reply app/tests/test_search.py]: this does not look correct, should not be two sets for column headers , also the column header should be 'Consignment reference' when ever used

File: app/templates/main/search-transferring-body.html [hunk 2]
```diff
@@ -26,121 +26,145 @@
...
-                                    {% endif %}
-                                </td>
+                                <th scope="col"
+                                    class="govuk-table__header govuk-table__header--search-header search__desktop-heading">
+                                    Series
+                                </th>
+                                <th scope="col"
+                                    class="govuk-table__header govuk-table__header--search-header search__mobile-heading">
+                                    Series / File name / Consignment ref
+                                </th>
```
Comment (pgandhizaizi) [inline app/templates/main/search-transferring-body.html:50]: why don't you introduce view type of something condition based to show the dynamically

**Review by anthonyhashemi — DISMISSED** [review]

**Comment by pgandhizaizi** [comment]
as discussed on the call with Terry and Colin, it has been agreed to perform separate test cases for desktop and mobile views and compare the single set of column headers at all the time

File: app/tests/test_search.py (hunk 1 above)
Comment (colinbowen) [reply app/tests/test_search.py]: @pgandhizaizi Updated to test for desktop column headers

File: app/templates/main/search-transferring-body.html (hunk 2 above)
Comment (colinbowen) [reply app/templates/main/search-transferring-body.html:50]: View_type cannot account for screen sizes.

Comment (pgandhizaizi) [reply app/templates/main/search-transferring-body.html:50]: after discussion it is clear , no need to implement view type

File: app/tests/test_search.py (hunk 1 above)
Comment (pgandhizaizi) [reply app/tests/test_search.py]: cleared the doubts after a call, approving the PR

**Review by pgandhizaizi — APPROVED** [review]

**Pull request merged**